tests/integration/helpers.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helpers for the glauth integration tests."""
+
+import logging
+import pathlib
+import zipfile
+from typing import Dict
+from urllib import request
+
+import toml
+
+logger = logging.getLogger(__name__)
+
+CONFIG = "sample-simple.cfg"
+CONFIG_URL = (
+    f"https://github.com/glauth/glauth/raw/0e7769ff841e096dbf0cb67768cbd2ab7142f6fb/v2/{CONFIG}"
+)
+ZIP = "config.zip"
+
+
+def _update_config() -> None:
```

Comment:
That's fair, I think that suggestion makes sense. I separated them initially in case we wanted to test other configurations and keep it modular but I think I'll leave that change to be made in the future if/when it's needed.

Reply:
Yeah... As I have learned over the past few months, it is best to avoid premature optimization. Maybe in the future, but we will let our needs several months from now decide that :sweat_smile: